src/app/log-in-seed/log-in-seed.component.html

```diff
@@ -35,6 +35,18 @@
     ></textarea>
   </div>
 
+  <div *ngIf="advancedOpen" class="main-text mb-20px">
+    If you have seed hex instead of seed phrase, enter it below.
```

Comment:
Change to "If you have a seed hex instead of a seed phrase, enter it below."

Minor grammar request!